### ISSUE TITLE
The results of retraining on kitti dataset is worse than the results produced by the pretrained model

### ISSUE THREAD

**Issue by Kanon777** [astra-vision/SceneRF#10]
I have followed the training pipeline and retrained the model on kitti dataset, but the result is worse than that produced by the pretrained model. I have tried several hyper-parameters(batch_size=4 lr=1.e-5, batch_size=8 lr=2e-5..), the results were still not comparable with the pretrained model.
So would you please provide some more suggestion of the training? Thanks a lot
The visual depth results are showed below
ours:
![image](https://user-images.githubusercontent.com/18673815/216274729-2e628a66-fe65-4f55-9564-55eb50e456f4.png)
pretrained model:
![image](https://user-images.githubusercontent.com/18673815/216275068-105dfb0a-f978-4c58-8e02-70fad0cf415d.png)

the detph eval metrics are showed below (the results are evaluated on the first 20 images)
ours:
|All     |0.206474|1.792310|7.127426|0.358133|0.642755|0.824265|0.903641|00000219|

pretrianed:
|All     |0.173536|1.199905|5.350147|0.273417|0.745632|0.901323|0.949406|00000219|

**Comment by anhquancao** [comment]
Hi @Kanon777, 
How many epochs did you train? As mentioned in the paper, the main model was trained for 50 epochs and the ablation ones was trained for 20 epochs.
Thus, the one I provided is trained for 50 epochs.

**Comment by Kanon777** [comment]
> 

i haved trained for 50 epochs

**Comment by anhquancao** [comment]
Hi, this is my training log, could you compare with it?
[Best4G8P_modelTypefull_lr1e-05_1200rays_b7_nGaus4_nPtsPerGaus8_std2.0_SOMSigma2.0_sphere1500x452_addfov20x8.zip](https://github.com/astra-vision/SceneRF/files/10567495/Best4G8P_modelTypefull_lr1e-05_1200rays_b7_nGaus4_nPtsPerGaus8_std2.0_SOMSigma2.0_sphere1500x452_addfov20x8.zip)

I also uploaded it here
https://tensorboard.dev/experiment/ixW80kKVT3ukiQdeZnZJ6g/#scalars

**Comment by Kanon777** [comment]
hi，
i have checked the log compared with mine, and found that the provided log seems stopped and resumed, and the loss of repoj was  descended when resumed training
![image](https://user-images.githubusercontent.com/18673815/217474740-04979508-4a1b-4510-b60c-c195fa322760.png)
![image](https://user-images.githubusercontent.com/18673815/217474807-e9e6d4b3-9fe6-4bcf-8cfd-4e51ddd734ff.png)

so was something changed when the training was resumed?

**Comment by anhquancao** [comment]
And how about other metrics evolve? Could you put your training metrics and the log in the same plot? 
I don't think it's the problem because even with 40k iterations, the metrics reach quite good number already, like ~5.8 RMSE.

**Comment by anhquancao** [comment]
Hi @Kanon777,

- I have just cloned the repository and retrained the SceneRF. The updated training results are highlighted in **blue** in the log, which you can view here:
https://tensorboard.dev/experiment/pLHrDjcqQkKxbeHNCXwf4w/#scalars&_smoothingWeight=0.932

- The jump you noticed in the log is simply due to a change in the logging:
![image](https://user-images.githubusercontent.com/9668858/228408742-3ed48413-e9c6-4da0-a86f-2a87a51bc77b.png)

- As for the other depth metrics such as RMSE and a1, they appear to be in good alignment, as shown in this image:
![image](https://user-images.githubusercontent.com/9668858/228408807-31635ed4-0514-45df-b7b9-c4860ae92697.png)
